style.css

```diff
@@ -0,0 +1,675 @@
+  align-self: center;
+  max-width: 500px;
+}
+
+.about-header-container .about-intro-paragraph {
+  border: 2px solid white;
+  background-color: #d3d3d3;
+  padding: 15px;
+  font-family: "Lato", sans-serif;
+  font-size: 12px;
+  font-weight: 400;
+  color: #4a4848;
+  margin-top: 30px;
+  text-align: center;
+  align-self: center;
+  max-width: 500px;
+}
+
+.about-header-container .about-contact-p {
+  font-family: "Cocogoose", sans-serif;
+  font-size: 10px;
+  color: #4a4848;
+  margin-top: 25px;
+  text-align: center;
+}
+
+.about-header-container .about-email {
+  font-family: "Lato", sans-serif;
+  font-size: 18px;
+  color: #4a4848;
+  margin-top: 25px;
+  text-align: center;
+  text-decoration: underline;
+}
+
+.logo-section {
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 30px 25px;
+}
+
+.logo-section .logo-section-head {
+  font-family: "Lato", sans-serif;
+  font-size: 28px;
+  color: #292828;
+  margin-top: 25px;
+  text-align: center;
+  max-width: 500px;
+}
+
+.logo-section .logo-section-desc {
+  font-family: "Lato", sans-serif;
+  font-size: 20px;
+  color: #4a4848;
+  margin-top: 25px;
+  text-align: center;
+  max-width: 500px;
+}
+
+.logo-section .logo {
+  margin-top: 32px;
+  border: 1px solid #2c2c2c;
+  max-width: 370px;
+  margin-bottom: 20px;
+}
+
+.past-summits-sections {
+  border: 1px solid black;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 20px 25px;
+  background-color: #fff;
+}
+
+.past-summits-sections .past-summits-heading {
+  margin-bottom: 15px;
+  font-family: "Lato", sans-serif;
+  font-style: normal;
+  color: #2c2c2c;
+  line-height: 28px;
+}
+
+.past-summits-sections .past-summits-desc {
+  font-family: "Lato", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 12px;
+  line-height: 21px;
+  color: #838383;
+  text-align: center;
+  margin-top: 15px;
+}
+
+.past-summits-sections .past-summits-images-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  align-self: center;
+  margin-top: 15px;
+}
+
+.past-summits-images-container .img-container .img-content-container {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.past-summits-images-container .img-container .img-content-container .summit-year {
+  font-family: "Lato", sans-serif;
+  font-size: 15px;
+  text-shadow: 1px 1px #363434;
+  margin-bottom: 10px;
+}
+
+.past-summits-images-container .img-container .img-content-container .summit-desc {
+  font-family: "Lato", sans-serif;
+  font-size: 8px;
+  text-shadow: 1px 2px #363434;
+}
+
+.partners {
+  display: flex;
+  flex-direction: column;
+  background-color: #414246;
+  padding-top: 30px;
+  padding-bottom: 30px;
+  align-items: center;
+  overflow: auto;
+}
+
+.partners .partners-heading {
+  color: white;
+  font-family: "Lato", sans-serif;
+  text-align: center;
+  margin-bottom: 12px;
+}
```

Comment:
- [ ] Kindly make sure to change the `Partner` heading color in order to match the Provided design.

   - ### YOUR DESIGN
   
   ![image](https://user-images.githubusercontent.com/87197412/225588230-10661e33-5b39-497c-836c-35d885e7c500.png)

   - ### BEHANCE DESIGN
   
     ![image](https://user-images.githubusercontent.com/87197412/225588297-f345fcf8-c6ea-4b7b-b27d-ebce4518cca1.png)